web/hawkHandler.go

```diff
@@ -132,7 +134,8 @@ func (h *HawkHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Step 6: *woot*, pass it on
-	h.handler.ServeHTTP(w, r)
+	reqCtx := r.WithContext(NewSessionContext(r.Context(), session))
```

Comment:
Is `NewSessionContext` cleaner than doing `reqCtx := r.WithContext(context.WithValue(ctx, sKey, session))` directly?